public/src/components/channelManagement/epicTests/epicTestEditor.tsx

```diff
@@ -8,6 +8,7 @@ import {
   DeviceType,
   SignedInStatus,
   PageContextTargeting,
+  ConsentStatus,
```

Comment:
we can edit it in the Epic editor, does it need to be added to the epic model?

Reply:
Yeah really good catch here! I've added it to the model for epics and headers but since the scope of this ticket is just for banners we have hidden the toggles from epics and headers but the functionality is there when we do decide to expand